functional-tests/test-bed/src/main/java/org/arquillian/smart/testing/ftest/testbed/project/ProjectBuilder.java

```diff
@@ -67,11 +67,10 @@ private BuiltProject executeGoals(String... goals) {
             : Paths.get(rootPath, POM_XML);
         final PomEquippedEmbeddedMaven embeddedMaven = EmbeddedMaven.forProject(pomFile.toString());
 
+        embeddedMaven.setDebugLoggerLevel();
```

Comment:
What is the benefit of having it enabled by default? It slows down the build doing quite a bit of I/O in the background (even though not visible unless an embedded build failed or specifically defined through test bed config).

Reply:
the debug log level is for MavenInvoker, not for the Maven build itself. 
The benefit is that it prints the whole command that is used for invoking the build. If you run it, there is almost nothing else printed so no slowdown is expected.
see: https://github.com/shrinkwrap/resolver#maven-build-invocation-command--logging

Reply:
Thanks for the reference. Naming got me confused. If so I'm totally fine with that.

Reply:
Yeah, the name can be confusing. I've explained the behavior in the Javadoc of this method.